// nodeutil/lib/mailutil.js
var nodemailer = require("nodemailer");
var os = require('os');
var smtpTransport = nodemailer.createTransport("SMTP",{
  host: "localhost"
});

var mailOptions = {
    from: os.hostname() , // sender address
    to: "receiver@mycompany.com", // list of receivers
    subject: "Hello Send From NMAIL", // Subject line
    text: "Dear:\n\n Hello, this is a sample mail. \n That you are success send this mail!\n]n Thanks.", // plaintext body
    html: "Dear:<br/><br/>Hello, this is a sample mail from <font color=red>nmail<font>. <br/>That you are success send this mail!  <br/><br/>Thanks." // html body
}

exports.sendNodeMail = function(receivers, subject, msg) {
    mailOptions.to = receivers;
    mailOptions.subject = subject;
    mailOptions.html = msg;
    smtpTransport.sendMail(mailOptions, function(error, response){
      if(error){
         console.log('[ERROR]%s',JSON.stringify(error));
      } else {
         console.log("Message sent: " + response.message);
      }
      smtpTransport.close(); // shut down the connection pool, no more messages
    });
}
